lib/organisms/widgets-area/widgets-area.directive.js

```diff
@@ -36,4 +38,23 @@ function lnOWidgetsArea($compile) {
 
     $compile(elem.contents())(scope);
   }
+
+  function getWidgetType(type) {
+    switch (type) {
+      case 'lean-downloads':
```

Comment:
Could we do this automatically, eg by removing the lean- prefix and replacing with ln-m-widget-